kentik_api_library/examples/demos/README.md

```diff
@@ -0,0 +1,29 @@
+# Running jupyter-notebook demos
+
+Precondition: python3 and pip3 are already installed.
+
+1. Install virtualenv - this allows you to create a python sandbox for your project  
```

Comment:
Nit: First couple of steps are duplicated with main README. But I understand this way it is easier to read (in one place)

Reply:
It seems that non-developers are going to play with the library, I wanted to provide comprehensive instruction so they dont get confused